packages/math-input/src/consts.ts

```diff
@@ -10,6 +12,11 @@ export const KeypadTypes = {
     EXPRESSION: "EXPRESSION",
 } as const;
 
+export type KeypadConfiguration = {
```

Comment:
Not needed in this PR, but I'd love to start moving these types to a `types.ts` file instead of them living on in `consts.ts`.  That feels not quite right. :)